taxonomy-manager-engine/src/main/java/com/digirati/taxman/analysis/nlp/corenlp/CoreNlpWordTokenizer.java

```diff
@@ -49,9 +53,13 @@ public static WordTokenizer create(String language) {
 
     @Override
     public List<WordToken> tokenize(String input) {
+        // Strip HTML tags and replace them with equivalent amount of '_' chars
+        HtmlTagReplacer.matcher(input).replaceAll(
```

Comment:
We should enable the clean XML annotator instead of doing this. https://stanfordnlp.github.io/CoreNLP/cleanxml.html

Reply:
Will it work with malformed XML (like HTML)? Does it preserve the character position?

Reply:
It should do. AFAIK it only searches for tags and doesn't try to parse it as a document.